backend/internal/interface/middleware/auth_token_version_test.go

```
@@ -0,0 +1,185 @@
+
+func newAuthEngine(tm *auth.TokenManager, lookup TokenVersionLookup) *gin.Engine {
+	gin.SetMode(gin.TestMode)
+	r := gin.New()
+	r.GET("/protected", Auth(tm, lookup), func(c *gin.Context) {
+		c.String(http.StatusOK, UserID(c))
+	})
+	return r
+}
+
+func request(r *gin.Engine, token string) *httptest.ResponseRecorder {
+	req := httptest.NewRequest(http.MethodGet, "/protected", nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)